net/strongswan/Makefile

```diff
@@ -527,8 +527,10 @@ define Package/strongswan-swanctl/install
 	$(INSTALL_DIR) $(1)/etc/swanctl/{bliss,conf.d,ecdsa,pkcs{12,8},private,pubkey,rsa}
 	$(INSTALL_DIR) $(1)/etc/swanctl/x509{,aa,ac,ca,crl,ocsp}
 	$(CP) $(PKG_INSTALL_DIR)/etc/swanctl/swanctl.conf $(1)/etc/swanctl/
+	echo "include /var/swanctl/swanctl.conf" >> $(1)/etc/swanctl/swanctl.conf
 	$(INSTALL_DIR) $(1)/usr/sbin
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/swanctl $(1)/usr/sbin/
+	$(INSTALL_BIN) ./files/swanctl.init $(1)/etc/init.d/swanctl
```

Comment:
Missing an $(INSTALL_DIR) $(1)/etc/init.d. Preferably add it at the top of the INSTALL macros of this package, so that they are somewhat in alphabetical order.

Reply:
Done.

Reply:
> Missing an $(INSTALL_DIR) $(1)/etc/init.d. Preferably add it at the top of the INSTALL macros of this package, so that they are somewhat in alphabetical order.

@stintel Please confirm that this has been done to your satisfaction.

Reply:
@stintel If all the changes you've requested have been made, please approve.